source/recurring_payments/index.html.md.erb

```diff
@@ -0,0 +1,389 @@
+---
+title: Take recurring payments
+last_reviewed_on: 2022-08-20
+review_in: 1 months
+weight: 50
+---
+
+# Take recurring payments
+
+GOV.UK Pay is still developing recurring payments so this feature is only available to a limited number of services at the moment. [Contact us](/support_contact_and_more_information/#during-office-hours) if you’re interested in taking recurring payments.
+
+If you are a beta service, you can take recurring payments from a user using GOV.UK Pay. The user consents to making recurring payments, enters into an ‘agreement’ with your service, and provides their payment details. The agreement allows you to take further payments through our API without the user interacting with your service again.
+
+Your service has [additional responsibilities when taking recurring payments](/recurring_payments/#1-understand-what-your-service-is-responsible-for). You’re responsible for:
+
+* authenticating users
+* getting and recording users’ consent for you to take recurring payments
+* managing payment schedules
+* telling users when you’re going to take payments
+* letting users amend their recurring payments
+
+This section outlines the recurring payment flow, your responsibilities, and how to set up and use agreements to take recurring payments.
+
+## How recurring payments work
+
+1. You create an ‘agreement’ between your service and your user through our API. This is an agreement for you to use your user’s payment details to make ongoing payments for your service.
+
+1. You set up the agreement by getting your user’s consent and taking their first payment. You instruct our API to save their payment details. This makes the agreement for recurring payments active.
+
+1. You take recurring payments by referencing the active agreement in your API requests when creating payments. The agreement allows GOV.UK Pay to authorise the payment with the user’s saved payment details.
+
+1. You cancel the agreement when it is no longer needed or when the user requests a cancellation. GOV.UK Pay cannot use the cancelled agreement to authorise payments. You can set the agreement up again by taking another payment and instructing our API to save the user’s payment details.
+
+You can [read more about the lifecycle of agreements](/recurring_payments/#understanding-agreement-status).
+
+### Understand the technical flow of recurring payments
+
+![A sequence diagram showing the journey of creating, setting up, and using an agreement to take recurring payments.](/images/recurring-payments-success-sequence.svg)
```

Comment:
The sequence suggests creating an agreement when the user would like to set up a recurring payment. 

There is another option where you create an agreement as soon as you have a new user who will be using your service, and then just set up the agreement if they want to do recurring payments.

I'm just thinking outloud for this, I think it's fine to show this a sequence and it's not saying it's the only prescriptive way it's just giving an example.

Reply:
I think that's a good point. I might add a qualifier in this section just saying something like 'In this integration example, the service creates an agreement when the user wants to set up a recurring payment'.

Hopefully that makes it a bit clearer that it's not the only way of doing it.